starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,84 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+.foLogo{
+  background-image: url(../images/soup.jpg);
+  background-size: cover;
+  height: 64vh;
+  width: 62vw;
+  margin: 0 auto;
+  text-align: center;
+  padding: 0;
+  background-position: center 100%;
+}
+
+.h1Style{
+  color: white;
+  font-size: 3.5em;
+  font-weight: 100;
+  padding-top: 3em;
+}
+
+.para1{
+  display: inline-block;
+  margin: 0 auto;
+  padding-left: 20em;
+  font-size: 20px;
+}
+
+hr{
+  width: 39em;
```

Comment:
same as the hr for the apple-pie recipe, give a little more width to the hr so that it is wider than the text. About 50em would have been good for this one.